apps/admin-portal/src/components/claims/add-edit-dialect.tsx

```diff
@@ -0,0 +1,150 @@
+/**
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the 'License'); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import React, { useEffect, useState } from "react";
+import { Modal } from "semantic-ui-react";
+import { Forms, Field, useTrigger, FormValue } from "@wso2is/forms";
+import { LinkButton, PrimaryButton } from "@wso2is/react-components";
+import { getADialect, updateADialect, addDialect } from "../../api";
+import { ClaimDialect, AlertLevels } from "../../models";
+import { useDispatch } from "react-redux";
+import { addAlert } from "../../store/actions";
+
+interface AddEditDialectPropsInterface {
+    open: boolean;
+    onClose: () => void;
+    update: () => void;
+    edit: boolean;
+    dialectID?: string;
+}
+export const AddEditDialect = (props: AddEditDialectPropsInterface): React.ReactElement => {
```

Comment:
Missing space between the interface and the component class. Fix other similar places as well when possible.